blackbox/docs/sql/statements/create-view.rst

```diff
@@ -0,0 +1,47 @@
+.. highlight:: psql
+.. _ref-create-view:
+
+===============
+``CREATE VIEW``
+===============
+
+Define a new view.
+
+.. rubric:: Table of Contents
+
+.. contents::
+    :local:
+
+Synopsis
+========
+
+::
+
+    CREATE [ OR REPLACE ] VIEW view_ident AS query
+
+
+Where ``query`` is a :ref:`SELECT statement <sql_reference_select>`.
+
+
+Description
+===========
+
+CREATE VIEW creates a named definition of a query. This name can be used in
```

Comment:
Shouldn't we add some restriction sentence, for such cases: https://github.com/crate/crate/pull/7104/commits/e420551b2eff2f394473f7bf48baf8a672a974df#diff-e3ea89595caa8377bb43a98d99e4cf49R74?

Reply:
yep good point

Reply:
Added a note https://github.com/crate/crate/pull/7104/commits/ad2b8f088f9e41adabe1a5701e4d1f0edf47b46b